app/Resources/TwigBundle/views/Exception/error503.html.twig

```diff
@@ -0,0 +1,11 @@
+{% extends '::_base.html.twig' %}
+
+
+{% block body %}
+    <h1>Oops! An Error Occurred</h1>
+    <h2>The server returned a "{{ status_code }} {{ status_text }}".</h2>
+
+    <div>
+        {{ exception.message }}
```

Comment:
Are you sure do you want to show the exception message? I've I'm not wrong, this could contain some information that should be only available for developers.

If I got it, on DEV environments we see the Symfony error page where we can read the exception message but in production we should only show that "something happened".